authority/linkedca.go

```diff
@@ -292,11 +292,22 @@ func (c *linkedCaClient) StoreRenewedCertificate(parent *x509.Certificate, fullc
 	return errors.Wrap(err, "error posting certificate")
 }
 
-func (c *linkedCaClient) StoreSSHCertificate(crt *ssh.Certificate) error {
+func (c *linkedCaClient) StoreSSHCertificate(prov provisioner.Interface, crt *ssh.Certificate) error {
 	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
 	defer cancel()
 	_, err := c.client.PostSSHCertificate(ctx, &linkedca.SSHCertificateRequest{
 		Certificate: string(ssh.MarshalAuthorizedKey(crt)),
+		Provisioner: createProvisionerIdentity(prov),
+	})
+	return errors.Wrap(err, "error posting ssh certificate")
+}
+
+func (c *linkedCaClient) StoreRenewedSSHCertificate(parent, crt *ssh.Certificate) error {
```

Comment:
Does it make sense to add the `provisioner.Interface` here too? Since the same `linkedca.SSHCertificateRequest` is used to post, I think the provisioner might be a useful piece of data to include? Or is the absence of that property meaningful somewhere different?

Reply:
That's something that I asked in #eng, right now we don't have access to it, and there are a few options of what we want to store, do we want to show SSHPOP? do we want to show the original provisioner?

Currently, I'm matching the X509 behavior, but as the renewal is different I might address this in a different PR.